### ISSUE TITLE
COMDLG_FILTERSPEC Missing from 0.52 was available in 0.48.

### ISSUE THREAD

**Issue by dtzxporter** [microsoft/windows-rs#2803]
### Summary

Shell::Common is no longer in windows_sys, which had `COMDLG_FILTERSPEC`.

### Crate manifest

```TOML
windows-sys = { version = "0.48", features = [
  "Win32_Foundation",
  "Win32_System_Com",
  "Win32_UI_Shell_Common",
  "Win32_UI_Shell",
  "Win32_UI_WindowsAndMessaging",
] }
```

### Crate code

```Rust
use Win32::UI::Shell::Common::COMDLG_FILTERSPEC;
```

**Comment by kennykerr** [comment]
This namespace appears to have been accidentally excluded from the `windows-sys` crate. Fixed here: #2804

**Comment by dtzxporter** [comment]
Thanks for the quick patch!
